app/assets/javascripts/app/views/comment_stream_view.js

```diff
@@ -27,6 +27,7 @@ app.views.CommentStream = app.views.Base.extend({
     this.model.comments.each(this.appendComment, this);
     this.commentBox = this.$(".comment_box");
     this.commentSubmitButton = this.$("input[name='commit']");
+    new app.views.CommentMention({el: this.$el, postId: this.model.get("id")});
```

Comment:
Please add a jasmine test checking that `app.views.CommentMention` is initialized here.